.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -745,7 +752,7 @@ stages:
       variables:
         TestAllPackageVersions: true
         IncludeMinorPackageVersions:  $[or(eq(variables.isMainBranch, 'true'), eq(variables.perform_comprehensive_testing, 'true'))]
-        publishTargetFramework: net5.0
+        publishTargetFramework: net6.0
```

Comment:
I think we need to convert to a matrix here, so we test arm64 on both .NET 5 and .NET 6

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/dacd82739691d921bad6981edab44a72b48e7824